app/views/equipment_models/_table.html.erb

```diff
@@ -2,25 +2,25 @@
   <thead>
   <tr>
     <th class="text-center col-md-2">Name</th>
-  <% if can? :manage, Reservation %>
-    <th class="no_sort text-center">Current Status</th>
-    <th class="text-center">Available</th>
-    <th class="text-center">Reserved</th>
-    <th class="text-center">Checked-Out</th>
-    <th class="text-center">Overdue</th>
-    <th class="text-center">Total Stock</th>
-    <% if can? :manage, EquipmentModel %>
-      <th class="no_sort text-center"> </th>
-      <th class="no_sort text-center"> </th>
-    <% end%>
-  <% end %>
+    <% if can? :view_all_dates, Reservation %>
```

Comment:
Did you add this permission to `app/models/ability.rb`? I haven't read the specs yet but I'm wondering how the authorization would work without it...